docker-compose.yml

```diff
@@ -56,15 +56,11 @@ services:
     restart: on-failure
     labels:
       traefik.enable: "true"
-      traefik.frontend.rule: "Host:keycloak.${PLATFORM_DOMAIN}"
```

Comment:
The whole idea is to have the development as close as possible to production, where the subdomains are preferable.